demo/src/main/java/com/orange/ods/demo/ui/components/utilities/ComponentCustomizationBottomSheet.kt

```diff
@@ -65,7 +63,15 @@ fun ComponentCustomizationBottomSheetScaffold(
                     }
                 },
                 text = stringResource(id = R.string.component_customize),
-                icon = { Icon(painter = painterResource(id = bottomSheetHeaderIconRes), contentDescription = null) })
+                icon = {
+                    Crossfade(targetState = bottomSheetScaffoldState.bottomSheetState.isExpanded) { expanded ->
```

Comment:
We could replace the crossfade animation with this one, I think it looks nicer:

```kotlin
val degrees = if (bottomSheetScaffoldState.bottomSheetState.isExpanded) 0f else -180f
val angle by animateFloatAsState(targetValue = degrees)
Icon(
    modifier = Modifier.rotate(angle),
    painter = painterResource(id = R.drawable.ic_chevron_down), contentDescription = null
)
```

In that case we can also remove the `ic_chevron_up.xml` resource.

Reply:
Top 👍